test/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/AtmosphericDrag.test.cpp

```diff
@@ -85,26 +86,12 @@ class OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_AtmosphericDrag : pu
    protected:
     void SetUp() override
     {
-        const Composite satelliteGeometry(Cuboid(
-            {0.0, 0.0, 0.0},
-            {Vector3d {1.0, 0.0, 0.0}, Vector3d {0.0, 1.0, 0.0}, Vector3d {0.0, 0.0, 1.0}},
-            {1.0, 2.0, 3.0}
-        ));
-
-        const Mass satelliteDryMass = Mass::Kilograms(100.0);
-        const Mass propellantMass = Mass::Kilograms(0.0);
-
-        satelliteSystem_ = {
-            satelliteDryMass,
-            satelliteGeometry,
-            Matrix3d::Identity(),
-            500.0,
-            2.1,
-        };
+        const Real mass = 100;
```

Comment:
Will be worth leveraging the `SatelliteSystem::Default()` that @vishwa2710 is adding in https://github.com/open-space-collective/open-space-toolkit-astrodynamics/pull/223.

Reply:
True, that will make things a little cleaner. Will add a todo for now to change over